### PR TITLE
M08 build test data

### PR DESCRIPTION
**Pull request by Sarosim**
## Description

This PR adds data to the following models:
### accounts app:
- CustomUser
### hackathon app
- Hackathon
- HackTeam
- HackProject
- HackProjectScoreCategory

## Pull request type
M08 build test data
- [X] #M01 (Miscellaneous User Story #01)
- [ ] #P02 (Participant User Story #02)
- [ ] #S03 (Staff User Story #03)
- [ ] #A04 (Admin User Story #04)
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Configuration instructions

profiles.json contains 20 users as per the screen shots below
`score_categories.json` contains the 5 Score Categories we use,
`hackathon#1.json` contains the events, projects, teams and the score categories as well

## Screenshots

Added 1 superuser on top of the existing admin
image

4 other staff, with no superuser status
image

14 participants:
image

Project Score Categories:
image

Projects (both belong to the same event):
image

Teams (belonging to two events):
image

and four events:
image

## Does this introduce a breaking change

- [ ] Yes
- [X] No

### CODE REVIEW

File: accounts/fixtures/profiles.json [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Postgres DATABASE_URL not found, using db.sqlite3
```
Comment (stefdworschak) [inline accounts/fixtures/profiles.json:1]: I think this file is not needed. I already added a file to `accounts/fixtures` to do this for one admin user and we probably don't need more.

File: hackathon/fixtures/hackathon#1.json [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Postgres DATABASE_URL not found, using db.sqlite3
```
Comment (stefdworschak) [inline hackathon/fixtures/hackathon#1.json:1]: Can you remove this line? Same for the other files.

Comment (stefdworschak) [inline hackathon/fixtures/hackathon#1.json:1]: Also, could you change the name of the file please? Just `hackathons` is fine.

**Review by stefdworschak — CHANGES_REQUESTED** [review]
@Sarosim the PR looks good. Could you please also add a command to the `scripts/seed.sh` and `scripts/seed.bat` files to seed these fixtures too?

File: accounts/fixtures/profiles.json (hunk 1 above)
Comment (stefdworschak) [reply accounts/fixtures/profiles.json:1]: Sorry ignore this please. Just saw that you added quite a lot, so let's leave these in! :+1:

**Comment by Sarosim** [comment]
All done, also removed the organisation part from my profiles fixture as it is done by `organisation.json`
score categories are also part of the `hackathons.json`, so I haven't included `score_categories.json` in the scripts, but can be useful to just load them separately.

**Review by stefdworschak — APPROVED** [review]
LGTM

**Pull request merged**